_layouts/pages/docs.html

```diff
@@ -1,8 +1,7 @@
 ---
 layout: page
 scripts:
-  - https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.js
-  - /js/docsearch.js
+  - "/js/build/documentation.js"
```

Comment:
Would be good for these file paths to have a cache breaker... Can we do that? You can use `assets-webpack-plugin` to write the cache breakers into a JSON file if you use a `filename` like `[name].js?[hash]`, could the site read that JSON file?